Resources/Audio/Effects/Emotes/attributions.yml

```diff
@@ -18,3 +18,8 @@
   license: "CC-BY-4.0"
   copyright: "Made by TV_LING. Converted from WAV to OGG and downmixed to 1 channel."
   source: "https://freesound.org/people/TV_LING/sounds/523467/"
+- files:
+    - salute.ogg
+  license: "CC-BY-SA-3.0"
+  copyright: "Taken from https://github.com/ss220-space/Paradise. No changes were made"
+  source: "https://github.com/ss220-space/Paradise/commit/89943a231ced9fc8db0aef65ff87093efee16b4e"
```

Comment:
Dead links.

Reply:
should I attach newer links for them?

Reply:
second link isnt dead, first link is just embedding badly with the period at the end, so just remove it